Failing unit test testParseArguments

One of the unit tests is currently failing:
```
$ python tests.py 
.....................F....Error getting manifest for non/existing:image
..........................
======================================================================
FAIL: testParseArguments (__main__.TestDEMain)
Tests the DockerExplorerTool.ParseArguments function.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "tests.py", line 99, in testParseArguments
    self.assertTrue(expected_usage in usage_string)
AssertionError: False is not true

----------------------------------------------------------------------
Ran 52 tests in 6.075s

FAILED (failures=1)
```
The test expects:
```
    expected_usage = (
        '[-h] [-d] [-r DOCKER_DIRECTORY] [-V]\n'
        '                {download,mount,list,history}')
```
But the actual output does not have a newline/indentation:
```
[-h] [-d] [-r DOCKER_DIRECTORY] [-V] {download,mount,list,history}
```

--- a/.gitignore
+++ b/.gitignore
@@ -22,3 +22,15 @@ debian/changelog
 # Test coverage files
 .coverage
 tests-coverage.txt
+
+# Environments
+.env
+.venv
+env/
+venv/
+ENV/
+env.bak/
+venv.bak/
+
+# Editor/IDE artifacts
+.vscode/

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,7 +1,7 @@
 dist: xenial
 language: python
 python:
-    - '3.7'
+    - '3.8'
 
 install:
     - python setup.py install

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,7 +1,8 @@
 dist: xenial
 language: python
 python:
-    - '3.8.5'
+    - '3.7'
+    - '3.8'
 
 install:
     - python setup.py install

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,7 +1,7 @@
 dist: xenial
 language: python
 python:
-    - '3.8'
+    - '3.8.5'
 
 install:
     - python setup.py install

--- a/docker_explorer/errors.py
+++ b/docker_explorer/errors.py
@@ -26,7 +26,7 @@ class DockerExplorerError(Exception):
     Args:
       message (str): the error message.
     """
-    super(DockerExplorerError, self).__init__(message)
+    super().__init__(message)
     self.message = message
 
 
@@ -47,7 +47,7 @@ class DownloaderException(DockerExplorerError):
     Args:
       message (str): the error message.
     """
-    super(DownloaderException, self).__init__(message)
+    super().__init__(message)
     self.message = message
     self.http_code = None
     self.http_message = None

--- a/tests.py
+++ b/tests.py
@@ -93,9 +93,10 @@ class TestDEMain(unittest.TestCase):
 
     options = de_object.ParseArguments()
     usage_string = de_object._argument_parser.format_usage()
-    expected_usage = (
-        '[-h] [-d] [-r DOCKER_DIRECTORY] [-V] {download,mount,list,history}')
+    expected_usage = '[-h] [-d] [-r DOCKER_DIRECTORY] [-V]'
+    expected_usage_commands = '{download,mount,list,history}'
     self.assertTrue(expected_usage in usage_string)
+    self.assertTrue(expected_usage_commands in usage_string)
     self.assertEqual(expected_docker_root, options.docker_directory)
 
   def testShowHistory(self):

--- a/tests.py
+++ b/tests.py
@@ -94,8 +94,7 @@ class TestDEMain(unittest.TestCase):
     options = de_object.ParseArguments()
     usage_string = de_object._argument_parser.format_usage()
     expected_usage = (
-        '[-h] [-d] [-r DOCKER_DIRECTORY] [-V]\n'
-        '                {download,mount,list,history}')
+        '[-h] [-d] [-r DOCKER_DIRECTORY] [-V] {download,mount,list,history}')
     self.assertTrue(expected_usage in usage_string)
     self.assertEqual(expected_docker_root, options.docker_directory)
 

